accounts/templates/accounts/profile_edit.html

```diff
@@ -0,0 +1,10 @@
+{% extends 'accounts/base.html' %}
+
+{% block title %}プロフィール編集{% endblock %}
+
+{% block content %}
+  <form method = "post">{% csrf_token %}
+    {{ form.as_p }}
+    <button type = "submit" name="プロフィール編集">編集完了</button>
```

Comment:
```suggestion
    <button type="submit" name="プロフィール編集">編集完了</button>
```
このように、= にスペースは入れないように統一してください。理由としては、通常HTMLの属性の=にスペースを入れることはないからです。他のところもバラバラなので統一お願いします。